### PR TITLE
added hover effects on the socials of users

### PR DESCRIPTION
**Pull request by Akanksha-202**
## Related Issue

In accordance to issue no. #638 

## Description

Added the hover effects as stated over the social links of project adders and over the GitHub icon in the navbar.

## Screenshots

image

[https://www.loom.com/share/e205b714b2ef43c29f046577c1ab1d7a](url)

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -4575,6 +4575,7 @@ packages:
       { integrity: sha512-+dQSyqPh4x1hlO1swXBiNb2HzTDN1I2IGLQx1GrBuiqFJfoMrnZWwVmatvSiO+Iz8fBUnf+lekwNo4c2LlXItg== }
     engines: { node: ">=14" }
     hasBin: true
+    dev: false
```
Comment (priyankarpal) [inline pnpm-lock.yaml:4578]: what is it?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml (hunk 1 above)
Comment (Akanksha-202) [reply pnpm-lock.yaml:4578]: I wasn't able to commit my changes due to some restrictions with prettier, so I had to install and check prettier cases 
And that made this change

Comment (priyankarpal) [reply pnpm-lock.yaml:4578]: like? what did you typed?

Comment (Akanksha-202) [reply pnpm-lock.yaml:4578]: i wrote 
pnpm install --save-dev prettier
pnpm prettier --write
pnpm prettier --check

these were the sol i got when i researched about the error.


Errors:
image
image

Comment (priyankarpal) [reply pnpm-lock.yaml:4578]: no no, don't do these mistakes in future. Please read contributing guidelines we already mentioned about this problem. if you face this type of warn related problem you need to format files using `pnpm format` after that stage all changes using `git add --all` then commit & push

Comment (Akanksha-202) [reply pnpm-lock.yaml:4578]: Ok I will redo

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**